src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -498,6 +498,32 @@ async Task<bool> OnPause(SessionId sessionId, JObject args, CancellationToken to
             {
                 var function_name = frame["functionName"]?.Value<string>();
                 var url = frame["url"]?.Value<string>();
+                if ("mono_wasm_add_lazy_load_files" == function_name)
+                {
+                    var loaded_assemblies = await SendMonoCommand(sessionId, MonoCommands.GetLazyLoadedFiles(), token);
+                    var res_val = loaded_assemblies.Value?["result"]?["value"];
+                    var lazy_loaded_files = res_val?.ToObject<string[]>();
+
+                    await
```

Comment:
IIUC, this is the same as the code at https://github.com/dotnet/runtime/blob/dbacdc2acbb9173a07a09777e4383160c5cc9c80/src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs#L791 . We should extract this to a method.

Reply:
Sure! I can make this change.